src/components/uikit/BottomSheet.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useCallback, useRef, ReactNode } from 'react';
+import { Dimensions } from 'react-native';
+import RNBottomSheet, {
+  BottomSheetProps as RNBottomSheetProps,
+} from '@gorhom/bottom-sheet';
+
+import { styled, useTheme } from '~styles';
+
+interface BottomSheetProps {
+  initialIndex: number;
+  snapPoints?: string[];
+  children: ReactNode;
+  onSheetChanges?: (index: number) => void;
+  keyboardBehavior?: RNBottomSheetProps['keyboardBehavior'];
+}
+
+export function BottomSheet({
+  initialIndex,
+  snapPoints = ['25%', '50%'],
```

Comment:
Imo there shouldn't be any snap points by default and they should be passed explicitly.

Reply:
Good point. I'll leave these as examples in an inline comment so people know what to use